File: go/1015.go
// Funciona no beecrowd
package main

import (
	"fmt"
	"math"
)

func main() {
	var x1 float64
	var y1 float64
	var x2 float64
	var y2 float64

	fmt.Scan(&x1)
	fmt.Scan(&y1)
	fmt.Scan(&x2)
	fmt.Scan(&y2)

	distancia := math.Sqrt(math.Pow(x2-x1, 2) + math.Pow(y2-y1, 2))
	fmt.Printf("%.4f\n", distancia)
}

File: Lua/Iniciante/1015.lua
-- Funcional no beecrowd

local A = io.read("n")
local B = io.read("n")
local C = io.read("n")
local D = io.read("n")

local E = math.sqrt((C - A)^2 + (D - B)^2)
print(string.format("%.4f", E))